app/herms.hs

```diff
@@ -504,11 +518,14 @@ optP t = subparser
                 (info  (helper <*> removeP t)
                        (progDesc (t Str.removeDesc)))
      <> command (t Str.shopping)
-                (info (helper <*> shopP t)
-                      (progDesc (t Str.shoppingDesc)))
+                 (info (helper <*> shopP t)
+                       (progDesc (t Str.shoppingDesc)))
```

Comment:
Did you intentionally change the spacing here? If so, why?

Reply:
Some of the other lines were misaligned, just wanted to maintain consistency.